dot.js

```diff
@@ -0,0 +1,23 @@
+let clicked=false;
```

Comment:
Instead of having a global variable here, we can check the state of the menu by checking if a class is present or no lets say "js-is-open".